src/main/java/com/karankumar/bookproject/ui/book/BookForm.java

```diff
@@ -158,47 +161,43 @@ private void closeForm() {
      * Binds the form fields and defines custom validators where necessary
      */
     private void bindFormFields() {
-        final String AFTER_TODAY_PREFIX = "The date you";
-        final String AFTER_TODAY_SUFFIX = "reading the book cannot be after today's date.";
-
         binder.forField(bookTitle)
-              .asRequired("Please provide a book title")
-              .bind(Book::getTitle, Book::setTitle);
+                .asRequired(BOOK_TITLE_ERROR)
```

Comment:
Could you ensure the period aligns with the previous method call (`forField()`)?

Reply:
Sorry for that, Intellij default formatting did that.

Reply:
That's fine. We have an [IntelliJ code style file](https://github.com/knjk04/book-project/blob/master/ide/intellij/book_project_code_style.xml) that you can import.